gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Fri Oct 05 14:15:04 CEST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.1-all.zip
```

Comment:
@thijslemmens  should we update the gradle wrapper version first in a separate PR ?

Reply:
ah, but we can't due to issue #24 that this PR actually tries to resolve ?